tests/phpunit/WC_Stripe_Helper_Test.php

```
@@ -1095,22 +1095,144 @@ public function test_build_line_items( bool $itemized = false, array $expected_i
+		$new_stripe_settings['capture']                    = 'yes';
+		$new_stripe_settings['pmc_enabled']                = 'yes';
+		WC_Stripe_Helper::update_main_stripe_settings( $new_stripe_settings );
+
+		update_option( \WC_Stripe_Feature_Flags::CHECKOUT_SESSIONS_FEATURE_FLAG_NAME, $feature_flag ? 'yes' : 'no' );
+
+		$is_checkout_filter = function () use ( $is_checkout ) {
+			return $is_checkout;
+		};
+		add_filter( 'woocommerce_is_checkout', $is_checkout_filter );
+
+		if ( 'subscription' === $cart_product_type ) {
```

PHPMD (2.15.0)

[error] The variable $cart_product_type is not named in camelCase. (undefined)

(CamelCaseVariableName)